plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -929,26 +930,32 @@ func (c *PodAffinityChecker) anyPodMatchesPodAffinityTerm(pod *v1.Pod, allPods [
 	if len(term.TopologyKey) == 0 {
 		return false, false, errors.New("Empty topologyKey is not allowed except for PreferredDuringScheduling pod anti-affinity")
 	}
-	matchingPodExists := false
+
 	namespaces := priorityutil.GetNamespacesFromPodAffinityTerm(pod, term)
 	selector, err := metav1.LabelSelectorAsSelector(term.LabelSelector)
 	if err != nil {
 		return false, false, err
 	}
+
+	podNum := int32(1)
 	for _, existingPod := range allPods {
 		match := priorityutil.PodMatchesTermsNamespaceAndSelector(existingPod, namespaces, selector)
 		if match {
-			matchingPodExists = true
```

Comment:
@k82cn Thanks! Based on what you just mentioned:
> podNum > term.MaxPodsPerTopologyKey is equal to matchingPodExists; according to your case (MaxPodsPerTopologyKey is 2), there's not enough matched pods, so it return false.

Shouldn't `MaxPodsPerTopologyKey` be actually called `MinPodsPerTopologyKey`? It looks like it refers the minimum number of pods we need in order to call this affinity a match.